src/custom-elements/src/components/progress/progress.tsx

```diff
@@ -0,0 +1,52 @@
+import { Component, Prop, Watch } from '@stencil/core';
+
+@Component({
+  tag: 'chi-progress',
+  styleUrl: 'progress.scss',
+  scoped: true
+})
+export class Progress {
+  /**
+   *  to set the value of a progress bar.
+   */
+  @Prop({ reflectToAttr: true }) value: string;
```

Comment:
Wouldn't it be better to use a number in this case to represent value, max and min?

Reply:
@SergioQCL I thought the same at first but within this component I'm not mutating the value so parsing it wasn't necessary so I went with string itself. Do you see something useful if we parse it?